.github/workflows/build.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         python-version: 3.9
     - run: pip3 install -U camkes-deps
     - run: sudo apt-get install doxygen
+    - run: sudo gem install bundler
```

Comment:
This can also be `sudo apt-get install bundler` if it's preferable to use the debian version

Reply:
I'm guessing, `apt` would give us a more stable version, but`apt` is super slow on those agents, so I slightly prefer this one if it doesn't produce any problems.